File: LEETCODE/Graphs/UniqueQuestions/997_findTheTownJudge/997_findTheTownJudge.py
class Solution:
    def findJudge(self, n: int, trust: List[List[int]]) -> int:
        outdegree=[0]*(n+1)
        indegree=[0]*(n+1)

        for t in trust:
            banda1,banda2=t[0],t[1]
            outdegree[banda1]+=1
            indegree[banda2]+=1

        for i in range(1,n+1):
            if indegree[i]==n-1 and outdegree[i]==0:
                return i
        
        return -1

File: LEETCODE/Graphs/UniqueQuestions/997_findTheTownJudge/997_findTheTownJudge.java
class Solution {
    public int findJudge(int n, int[][] trust) {
        int outdegree[]= new int[n+1];
        int indegree[]= new int[n+1];
        for(int t[]:trust){
            int banda1= t[0];
            int banda2= t[1];
            outdegree[banda1]++;
            indegree[banda2]++;
        }
        for(int i=1;i<=n;i++){
            if(indegree[i]==n-1 && outdegree[i]==0){
                return i;
            }
        }
        //
        return -1;
    }
}